.github/steps/3-step.md

```diff
@@ -52,7 +49,7 @@ Now, let's use our MCP server's tools and Copilot to do a bit of triage and get
    > #codebase Let's do the first one. Follow these steps:
    > 1. Checkout a new local branch for making our changes.
    > 2. Make the changes then confirm with me that they look correct.
-   > 3. Push the changes and start a pull request.
+   > 3. Push the changes and create a pull request.
```

Comment:
The tool is called `#create_pull_request` - this increases the chances of it being called instead of `gh pr` cli which happened to me